pyproject.toml

```diff
@@ -30,17 +30,20 @@ requires-python = ">=3.10"
 # Run 'tox -e deps' after making changes here. This will update requirement files.
 # Make sure to list one dependency per line.
 dependencies = [
-  "rich",
-  "essreduce>=24.10.3",
   "confluent_kafka",
-  "ess-streaming-data-types"
+  "essreduce>=24.10.3",
+  "ess-streaming-data-types",
+  "msgpack",
+  "rich",
+  "zmq",
 ]
 
 dynamic = ["version"]
 
 [project.scripts]
 beamlime = "beamlime.__main__:main"
 beamlime-raw-detectors = "beamlime.executables.show_detector:main"
```

Comment:
once https://github.com/scipp/beamlime/pull/272 is merged the project script section should be updated.